test/jdk/ProblemList.txt

```diff
@@ -890,3 +890,34 @@ jdk/jfr/event/os/TestThreadContextSwitches.java                 8247776 windows-
 tools/jpackage/share/jdk/jpackage/tests/ModulePathTest3.java#id0    8248418 generic-all
 
 ############################################################################
+
+# Client manual tests
+
+java/awt/event/MouseEvent/SpuriousExitEnter/SpuriousExitEnter_1.java 7131438,8022516 generic-all
+java/awt/event/MouseEvent/SpuriousExitEnter/SpuriousExitEnter_2.java 7131438,8022516 generic-all
+java/awt/Modal/WsDisabledStyle/CloseBlocker/CloseBlocker.java 7187741 generic-al
+java/awt/Component/UpdatingBootTime/UpdatingBootTime.html 7194219 linux-all
+java/awt/xembed/server/TestXEmbedServerJava.java 8001150,8004031 generic-all
+java/awt/TextArea/MouseOverScrollbarWhenTyping/Test1.java 8005970 macosx-all,windows-all
+javax/swing/JFileChooser/6698013/bug6698013.java 8024419 macosx-all
+java/awt/Focus/8044614/bug8044614.java 8029261 generic-all
+javax/swing/JColorChooser/8065098/bug8065098.java 8065647 macosx-all
+java/awt/Modal/PrintDialogsTest/PrintDialogsTest.java 8068378 generic-all
+java/awt/dnd/DnDFileGroupDescriptor/DnDFileGroupDescriptor.html 8080185 macosx-all,linux-all
+javax/swing/JTabbedPane/4666224/bug4666224.html 8144124  macosx-all
+java/awt/event/MouseEvent/AltGraphModifierTest/AltGraphModifierTest.java 8162380 generic-all
+java/awt/Window/TranslucentJAppletTest/TranslucentJAppletTest.java 8163086 macosx-all
+java/awt/image/multiresolution/MultiResolutionIcon/IconTest.java 8166522,8250804 macosx-all,linux-all
+java/awt/image/VolatileImage/VolatileImageConfigurationTest.java 8171069 macosx-all,linux-all
+java/awt/Modal/InvisibleParentTest/InvisibleParentTest.java 8172245 linux-all
+java/awt/print/Dialog/RestoreActiveWindowTest/RestoreActiveWindowTest.java 8185429 macosx-all
+java/awt/TrayIcon/DblClickActionEventTest/DblClickActionEventTest.html 8203867 macosx-all
+java/awt/Frame/FrameStateTest/FrameStateTest.html 8203920 macosx-all,linux-all
+javax/swing/SwingUtilities/TestTextPosInPrint.java 8227025 windows-all,linux-all
+java/awt/print/PrinterJob/ScaledText/ScaledText.java 8231226 macosx-all
+java/awt/print/bug8023392/bug8023392.html 8231231 generic-all
+java/awt/font/TextLayout/TestJustification.html 8250791 macosx-all
```

Comment:
The JDK-8250791 initially was reported on Ubuntu, not sure should add it here or not.